.travis.yml

```diff
@@ -1,4 +1,27 @@
+---
+language: python
+dist: focal
 git:
   submodules: true
-script:
+before_install:
   - repo_checks/submodule_check.sh
+install:
+  - git submodule update --remote
+  - .travis/install.sh
+script:
+  - .travis/script.sh $TEST
+jobs:
+  include:
+  # - env: TEST=pulp2-nightly-pulp3-source-centos7 # Not enough RAM
```

Comment:
😢

Reply:
I'm more than happy to try to get it working, and I believe I can since I think it's just a matter of RAM/swap for me to adjust. I just want to get this reviewed (feedback on design / implementation) and reviewed 1st.